### PR TITLE
Backoffice: Series sequence relations

### PR DESCRIPTION
**Pull request by kpsherva**
Screenshot 2020-02-18 at 11 39 24
Screenshot 2020-02-18 at 11 39 30
Screenshot 2020-02-18 at 17 36 45
Screenshot 2020-02-18 at 17 56 09

### CODE REVIEW

**Review by ntarocco — COMMENTED** [review]
It does not show this field in the frontsite right?

**Comment by kpsherva** [comment]
> It does not show this field in the frontsite right?

no, I will add it separately to the issue when someone fixes the frontsite series detail page

File: ui/src/pages/backoffice/Series/SeriesDetails/components/SeriesDocuments/SeriesDocuments.js [hunk 1]
```diff
@@ -64,25 +64,24 @@ export default class SeriesDocuments extends Component {
...
+            data={seriesDocuments.hits}
+            columns={columns}
+            totalHitsCount={seriesDocuments.total}
+            name={'documents'}
+            seeAllComponent={this.seeAllButton()}
+            showMaxRows={showMaxDocuments}
+            renderEmptyResultsElement={() => (
+              <InfoMessage
+                header="No volumes in this series."
+                content="Start with a book/article to attach it to this series."
```
Comment (ntarocco) [inline ui/src/pages/backoffice/Series/SeriesDetails/components/SeriesDocuments/SeriesDocuments.js]: minor: WDYT with: start **from** a ...

**Review by ntarocco — APPROVED** [review]

File: ui/src/api/utils.js [hunk 2]
```diff
@@ -63,6 +63,33 @@ export const schemaToPidType = schema => {
...
+  } else if (extraRelationField.type === 'previous') {
+    return {
+      next_pid: referer.metadata.pid,
+      next_pid_type: recordToPidType(referer),
+      previous_pid: selectedRelated.metadata.pid,
+      previous_pid_type: recordToPidType(selectedRelated),
+      relation_type: 'sequence',
+    };
+  } else {
+    throw TypeError('Wrong sequence type. Should be prev or next.');
```
Comment (topless) [inline ui/src/api/utils.js]: previous instead of prev

File: ui/src/api/utils.js [hunk 3]
```diff
@@ -63,6 +63,33 @@ export const schemaToPidType = schema => {
   }
 };
 
+export const sequenceRelationPayload = (
+  referer,
+  selectedRelated,
```
Comment (topless) [inline ui/src/api/utils.js]: Do these terms `referrer`, and `seletedRelated`  pro exist? (for example from the generic component you have implemented) if not I would keep the same `previous`, `next`.

nit: referrer has a typo, needs double rr

File: ui/src/pages/backoffice/Series/SeriesDetails/components/SeriesRelations/RelationSequence/RelationSequence.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+            <Grid.Column width={8}>
+              <Divider horizontal> Continuations of this serial </Divider>
+              <ExistingRelations
+                rows={next}
+                showMaxRows={showMaxRows}
+                columns={columns}
+                emptyMessage={
+                  <InfoMessage
+                    header={'No continuations'}
+                    content={'Use the button above to sequence relations.'}
```
Comment (topless) [inline ui/src/pages/backoffice/Series/SeriesDetails/components/SeriesRelations/RelationSequence/RelationSequence.js]: Sequence is not verb, maybe something like "Use the button above to create a sequential relation"

**Review by topless — APPROVED** [review]

File: ui/src/pages/backoffice/Series/SeriesDetails/components/SeriesDocuments/SeriesDocuments.js (hunk 1 above)
Comment (kpsherva) [reply ui/src/pages/backoffice/Series/SeriesDetails/components/SeriesDocuments/SeriesDocuments.js]: 👍 maybe you have an idea for something more specific. I wanted to express to start from book/article **details page** but I'm not sure if it is clear for the user

File: ui/src/pages/backoffice/Series/SeriesDetails/components/SeriesRelations/RelationSequence/RelationSequence.js (hunk 4 above)
Comment (kpsherva) [reply ui/src/pages/backoffice/Series/SeriesDetails/components/SeriesRelations/RelationSequence/RelationSequence.js]: 👀  thanks I ate the word

**Pull request merged**